client/state/site-settings/exporter/actions.js

```diff
@@ -2,16 +2,33 @@
  * Internal dependencies
  */
 import notices from 'notices';
+import wpcom from 'lib/wp';
+import debugModule from 'debug';
 import i18n from 'lib/mixins/i18n';
 
+import { prepareExportRequest, getDataState } from './selectors';
+import { getSelectedSite } from 'state/ui/selectors';
+
+const debug = debugModule( 'calypso:exporter' );
+const wpcomUndocumented = wpcom.undocumented();
+
 import {
 	SET_EXPORT_POST_TYPE,
+	SET_EXPORTER_ADVANCED_SETTING,
+
+	FETCH_EXPORTER_ADVANCED_SETTINGS,
+	RECEIVE_EXPORTER_ADVANCED_SETTINGS,
+
 	REQUEST_START_EXPORT,
 	REPLY_START_EXPORT,
 	FAIL_EXPORT,
 	COMPLETE_EXPORT
 } from 'state/action-types';
 
+const ERROR_START_EXPORT = i18n.translate( 'We had a problem starting your export. Please check your internet connection and try again.' );
+const ERROR_POLLING = i18n.translate( 'We had a problem checking the status of your export. Please check your internet connection and try again.' );
+const ERROR_PROCESSING = i18n.translate( 'We had a problem processing your export. Please contact support.' );
```

Comment:
I'm a bit suspicious about storing error messages here in `actions.js`

For one, I don't believe that these translations will update, because these lines of code will probably only run when the module loads, whereas if the user switches languages, the translations won't change because that code is out of the control flow. When we put strings in `render()` they get updated properly.

Second, it feels like mixing a view item with an action item. Ideally, we could still choose to use these constants, but instead of adding the rendered text here, we could just leave them as (meaningful) value-less constants and map the constants to their intended meaning at render time.

I'm not sure what would lead me here if I had discovered a bug in the messages. I would be looking through the React components to try and find them.